machine_management/user_infra/adding-vsphere-compute-user-infra.adoc

```diff
@@ -11,6 +11,13 @@ You can add more compute machines to your {product-title} cluster on VMware vSph
 
 * You xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[installed a cluster on vSphere].
 
+* You have installation media and {op-system-first} images that you used to create your cluster. If you do not have these files, you must obtain them by following the instructions in the xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[installation procedure].
```

Comment:
This prereq was not in the vSphere doc before. I am not sure if it is necessary to include this with the note about adding more compute machines. @miabbott please confirm if this should remain.

Reply:
Until we have the ability to generate boot images in the cluster (https://github.com/openshift/enhancements/pull/201), we still have this requirement.